chingsley/ireporter
# ft-fetch-signup-#163056237 implement fetch to consume endpoint for us…

#### What does this PR do?
This pull request deals with implementation of user sign up, by using fetch to consume the API endpoint.

#### Description of Task to be completed?
- create the register.js file to implement the fetch
- using fetch consume the API with the route [host]/api./v1/auth/signup
- Display error messages by calling the 'showDialogMsg()' function with the right arguments
- on successful registration, display a success message and suggest to the user to sign in and start reporting cases.

#### How should this be manually tested?
- clone the repo
- cd into the root directory
- run `npm install' to install the necessary dependencies
- run `npm test' to see the tests pass
- run `npm start' to start the server
- at the frontend, test the functionalities by trying to sign-up a new user, providing both correct and incorrect credentials

#### What are the relevant pivotal tracker stories
#163056237
